src/core/public/application/capabilities/capabilities_service.mock.ts

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 import { CapabilitiesService, CapabilitiesStart } from './capabilities_service';
-import { deepFreeze } from '../../utils/deep_freeze';
+import { deepFreeze } from '../../../utils/';
```

Comment:
Remove trailing slash for import consistency.

Reply:
let's introduce linter rule explicitly if it's important. imo it doesn't make sense to perform manual check on every PR
related https://github.com/elastic/kibana/pull/36829#discussion_r289798606